### ISSUE TITLE
Add possibility for color per entry

### ISSUE THREAD

**Issue by juxeii** [petereon/beaupy#54]
Would it be possible to have an option for colors per entry in lets say multi-select?
It would be perfect to show the user critical/important choices(e.g. in red).
So, instead of just providing a list of options, would it be possible to provide a list of tuples, with first entry the text, second entry the color of the text? 
If no color is given, just use the default one.

Thx for this great module!

**Comment by petereon** [comment]
Hi @juxeii , you can color and style options using rich syntax, it is [partly demonstrated in the example](https://github.com/petereon/beaupy/blob/5f61b3e6f72866cf73b9dadb451f4ae5e1bb4c16/example.py#L17). I have included the relevant excerpt extended with a bit more styling below.

```python
names = [
      "Frodo [bold]Baggins[/bold]",
      "[#ff0000]Samwise Gamgee[/#ff0000]",
      "Legolas",
      "Aragorn",
      "[blue]Sauron[/blue]",
  ]
  name = select(names, cursor=">", cursor_style="cyan")
```

I am aware it requires a bit of manual work to massage the strings, but I would not really like to have tuples as I want to have `beaupy` dead-simple.

**Comment by juxeii** [comment]
Thanks, did not know that!
Works nicely.

**Comment by juxeii** [comment]
I have an issue with the colors in `select_multiple`.
I color some options, for example:

![image](https://user-images.githubusercontent.com/4497675/231579176-827cfd6a-6605-46f0-b9d8-0440e116eb1d.png)
The issue is that I cannot see the current active selection, which I specified with `cursor_style='green'`

Now, when I do not color the options, like so:

![image](https://user-images.githubusercontent.com/4497675/231579829-cb580d2c-bc97-4efc-bebd-d8b78c1a76b3.png)
The cursor style is now back and the users see the current selection.

Am I doing something wrong here? No matter which color I choose for the options, as soon as they are colored, I can no longer see the selection.

**Comment by petereon** [comment]
Thanks for reporting, this is a good catch. It's a trivial issue but I can't think of an elegant fix. Let me mull this over for a bit, I'll probably have it resolved during the weekend.

**Comment by petereon** [comment]
There is now a pre-release [`3.5.3b0`](https://github.com/petereon/beaupy/releases/tag/v3.5.3b0) that resolves this, [also available on PyPI](https://pypi.org/project/beaupy/3.5.3b0/)

**Comment by juxeii** [comment]
Hi, thx. Seems to work.

**Comment by petereon** [comment]
Released this as full release (`3.5.3`) today.
